examples/seq2seq_attn/seq2seq_attn.py

```diff
@@ -100,11 +100,12 @@ def forward(self, batch, mode):
         else:
             start_tokens = torch.ones_like(batch['target_length']) * \
                            self.bos_token_id
+
             helper_infer = self.decoder.create_helper(
                 decoding_strategy="infer_greedy",
                 embedding=self.target_embedder,
                 start_tokens=start_tokens,
-                end_token=start_tokens.new_tensor(self.eos_token_id))
```

Comment:
I might be wrong, but isn't `self.eos_token_id` already a Python int? Could you test run this to make sure it works?

Reply:
`self.eos_token_id` is a `numpy.int64`. If we use `self.eos_token_id` directly, we will get an error here (https://github.com/asyml/texar-pytorch/blob/master/texar/modules/decoders/decoder_helpers.py#L247).

Reply:
I see, thanks for clarifying!